### ISSUE TITLE
Translation List is too long

### ISSUE THREAD

**Issue by nickhamze** [WordPress/wporg-theme-directory#101]
For some of these themes the list of translations is bonkers. What do you think about a "read more" button that hides everything but the first 10 or so.
<img width="380" alt="Screenshot 2024-05-30 at 5 41 09 PM" src="https://github.com/WordPress/wporg-theme-directory/assets/158918/4f5b24ce-212f-478b-ab82-c90c233ae546">

**Comment by ryelle** [comment]
The Twenty* & most-popular themes are outliers in terms of translations.

> For a little more reference, while Twenty Nineteen is translated into 63 languages, only 25 themes have >50 languages, so it is an extreme case. Most themes that have translations have <5.
> (discussed on https://github.com/WordPress/wporg-theme-directory/pull/72)

I would also argue that many translations is a sign of a "good" theme (it's got enough support or community use to be translated).

I still think we can wait on wider feedback about this before building something, but I'll leave this issue open to collect that feedback.

**Comment by nickhamze** [comment]
If that 50+ translations is a sign of theme health than over 12k themes are very very sick :) .

The vast majority of the people looking at themes aren't looking for these translations. Adding all this noise (the current version just has a link) seems strange.

**Comment by ryelle** [comment]
> If that 50+ translations is a sign of theme health than over 12k themes are very very sick

I mean, it's one data point in many when choosing a theme.

> the current version just has a link

That's true, but adding the list was in response to meta requests: #28/[#5925-meta](https://meta.trac.wordpress.org/ticket/5925), [#1383-meta](https://meta.trac.wordpress.org/ticket/1383).

**Comment by nickhamze** [comment]
Is there a reason you didn't display it like it does in the plugin repo? That actually looks and works quite nice.

<img width="403" alt="Screenshot 2024-05-31 at 10 26 48 AM" src="https://github.com/WordPress/wporg-theme-directory/assets/158918/fac4ac61-884c-4826-8008-f1c3b8663cca">

**Comment by nickhamze** [comment]
I also like that it puts a number to it (See all 7). You can see at a glance see how many languages it's translated into and use that as a signal if the theme is widely supported.

**Comment by ryelle** [comment]
It's not a shared component, so for Themes it would need to be rebuilt. As I said before, I didn't think it was necessary since so few themes have many translations. That's why I've left this open for more feedback.

**Comment by StevenDufresne** [comment]
I agree that it probably makes sense to move it up into the meta details to match plugins. 

While not many themes do have translations, the ones that receive the most traffic have many and the translation block can be distracting.

**Comment by timse201** [comment]
I think it makes sense that it matches with plugin dir.
How about using the browser language header and placing these languages and their variants / similar languages above all others.

e.g.
Your languages:
Español de Argentina, Español de Chile, Español de Costa Rica, Español

others:
...
